services/madoc-ts/src/frontend/shared/capture-models/editor/input-types/AutocompleteField/AutocompleteField.tsx

```diff
@@ -34,25 +33,26 @@ export type CompletionItem = {
 
 function renderOptionLabel(option: CompletionItem) {
   return (
-    <>
-      <LocaleString as="strong" style={{ lineHeight: '1.8em', verticalAlign: 'middle' }}>
-        {option.label as any}
-      </LocaleString>
-      {option.resource_class ? <Tag style={{ float: 'right', marginLeft: 10 }}>{option.resource_class}</Tag> : null}
-    </>
+    <div style={{ display: 'flex', flexDirection: 'column' }}>
+      <div>
+        <LocaleString as="strong" style={{ lineHeight: '1.8em', verticalAlign: 'middle' }}>
+          {option.label as any}
+        </LocaleString>
+        {option.resource_class ? <Tag style={{ float: 'right', marginLeft: 10 }}>{option.resource_class}</Tag> : null}
+      </div>
+      {option.description && <LocaleString>{option.description as any}</LocaleString>}
+    </div>
   );
 }
 
 export const AutocompleteField: FieldComponent<AutocompleteFieldProps> = props => {
   const { t } = useTranslation();
-  const [options, setOptions] = useState<CompletionItem[]>(
-    props.value ? [typeof props.value === 'string' ? { uri: props.value, label: 'unknown' } : props.value] : []
```

Comment:
this line was the cause of the catch error 

`syntaxerror: JSON.parse: unexpected character at line 1 column 1 of the JSON data`

Reply:
Hmm I think `props.value` could still be a string in some capture model cases.